### ISSUE TITLE
NullPointerException in com.github.tomakehurst.wiremock.http.ContentTypeHeader

### ISSUE THREAD

**Issue by bertm** [tomakehurst/wiremock#643]
I'm writing a customer matcher that uses the request's Content-Type internally. For this I envision to use the  `ContentTypeHeader`'s method `mimeTypePart()`.
When the Content-Type header is not set however, this method throws a `NullPointerException` because the `parts` member variable remains `null` and is dereferenced in this method.

Minimal example:
```java
ContentTypeHeader header = ContentTypeHeader.absent();
header.mimeTypePart(); // throws NullPointerException internally
```

Expected:
```java
ContentTypeHeader header = ContentTypeHeader.absent();
header.mimeTypePart(); // returns null or ""
```

**Comment by tomakehurst** [comment]
Thanks for reporting this. Feel like raising a PR with a fix?

**Comment by tomakehurst** [comment]
Fixed by 77743d03594da1b00d83838b29ffb21aea4b9bf2
